src/DatabaseOutputDriver.C

```diff
@@ -300,7 +303,10 @@ void DatabaseOutputDriver::writeSQLLog() {
        long size = ftell(fh);
        fseek(fh, 0, SEEK_SET);
        char *buffer = new char[size + 1];
-       fread(buffer, sizeof(char), size, fh);
+       if(fread(buffer, sizeof(char), size, fh) != size) {
```

Comment:
So up there you changed the buffer to a vector of char, do you want to do it here as well?